src/ontology/mondo-ingest.Makefile

```diff
@@ -13,6 +13,28 @@
 MAPPINGSDIR=../mappings
 SKIP_HUGE=false
 
+####################################
+### Package management #############
+####################################
+.PHONY: pip
+pip:
+	python3 -m pip install --upgrade pip setuptools
+
+.PHONY: oaklib
+oaklib: pip
+	 python3 -m pip install --upgrade $@
+
+.PHONY: sssom
+sssom: pip
+	python3 -m pip install --upgrade $@
+
+.PHONY: semsql
+semsql: pip
+	python3 -m pip install --upgrade semsql>=0.3.2
```

Comment:
@hrshdhgd Previously was `pip install --upgrade semsql==0.3.2`. Do we need to lock into that version specifically? Or can we just do `semsql` or `semsql>=0.3.2`?

Reply:
I dont remember, sorry.

Reply:
Latest should suffice.

Reply:
Makes sense to me. Changed!